rllib/trainer/pg/pg_torch_policy.py

```diff
@@ -0,0 +1,86 @@
+"""
```

Comment:
Simply moved here, no changes done.

Reply:
are there still things in agents/ directory if we move the policies etc here?
personally, I feel like the existing structure of having all the custom overrides in agents/ felt pretty natural. this also minimizes disruptive changes for users. basically:

```
rllib/
  - trainer/   # generic trainer definitions
  - policy/    # generic policy definitions
  - evaluation/   # worker definitions
  - agents/
     - pg/    # pg overrides of policy and trainer
     - other agents/
```

as written, having policy overrides under trainer/ feels slightly illogical.

Reply:
I see your point. However, we are already doing this illogically today: We define policy-overrides inside the `agents` dir.
The reason of starting to rename `agents` into `trainer` with this PR is that the word "agent" should only be used for acting entities in a multi-agent/single-agent environment, not for the "thing that trains policies". Our Trainers - e.g. PPOTrainer - used to be called Agents - e.g. "PPOAgent" - and we renamed these classes some time ago w/o renaming the directory at the same time. I would like to start moving everything from "agents" into "trainer" and then remove the agents dir entirely. But yes, maybe we should do this all at once (and not in this PR) or one by one. Not sure.

On the policy overrides: Would it be better to move these into the `policy` dir? I'm not sure this would be a good idea.

```
rllib/
  trainer/  # all contents of `agents` will move into here, eventually
    trainer.py
    pg/
      pg_trainer.py
      pg_tf_policy.py
      pg_torch_policy.py
    ppo/
      ...
    a3c/
      ...
  agents/  # deprecate soon: b/c confusing terminology, which clashes with single-agent/multi-agent envs
  policy/  # generic policy defs (remain as-is)
  
```

Reply:
Let me know, what you think.